dotnet/samples/Concepts/Prompty/PromptyFunction.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
nit: Same comment to update README with link to this file:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/README.md

Reply:
Follow up task https://github.com/microsoft/semantic-kernel/issues/6166